gaphor/plugins/autolayout/pydot_patch.py

```diff
@@ -0,0 +1,69 @@
+# A workaround for Pydot, to make sure it works
+# in a PyInstaller setup.
+# The application path is updated and the
+# environment variable VGBINDIR is set to make sure plugins are found
+
```

Comment:
Would others using pydot with PyInstaller find this useful as well? Could we put it in a run-time hook for PyInstaller?

Reply:
I haven't checked with the pydot authors. There has not been much activity for the past year. Maybe I should switch to [pygraphviz](https://github.com/pygraphviz/pygraphviz)? We still need our custom hook then, due to conflicting pango and cairo libraries.

Pydot is 100% Python, which should be easier for our multi-platform use case, though.